src/components/BaseLayerSwitcher/BaseLayerSwitcher.js

```diff
@@ -107,7 +107,7 @@ function BaseLayerSwitcher({
     : baseLayers.map((layer) => layer.get('previewImage'));
 
   const openClass = switcherOpen ? ' rs-open' : '';
-  const closedClass = isClosed ? ' rs-closed' : '';
+  const hiddenStyle = switcherOpen && !isClosed ? 'visible' : 'hidden';
```

Comment:
use rs prefix pls

Reply:
Nope, these are not classes, but strings applied to the style object used in react, so adding a prefix will break it